pkg/kubelet/container/export_test.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package container
```

Comment:
I'm not sure I understand the question, but yes - this file is here to break circular dependencies. The container tests need to be in a separate package (container_test) because the use `container/testing` files, which in turn depend on `container`. `runtimeCache` is not exported (nor are it's `updateCache` and `pods` methods), so this file "exports" them to the tests. BTW - this follows the pattern used in the go library (e.g. https://golang.org/src/net/http/export_test.go)